app/mailers/invitation_mailer.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class InvitationMailer < ActionMailer::Base
+  DEFAULT_INVITATION_SUBJECT = 'u-can-act vragenlijst onderzoek'
+  layout 'mailer'
+
+  default from: ENV['FROM_EMAIL_ADDRESS']
+
+  def invitation_mail(email_address, message, invitation_url)
+    @invitation_url = invitation_url
+    @message = message
+    mail(subject: DEFAULT_INVITATION_SUBJECT, to: email_address)
```

Comment:
Gaat het goed als we altijd hetzelfde onderwerp gebruiken? Of willen we bijv het weeknummer in de titel zetten?

En moet het onderwerp dan niet iets zijn als "u-can-act: Vul nu de wekelijkse vragenlijsten in" oid ipv "u-can-act vragenlijstonderzoek"? (wat trouwens één woord is).

Of willen we juist de dynamische regel in het subject hebben? bv: "u-can-act: Hoi Koos ff invullen die hap"

Reply:
> Gaat het goed als we altijd hetzelfde onderwerp gebruiken? Of willen we bijv het weeknummer in de titel zetten?
>
> En moet het onderwerp dan niet iets zijn als "u-can-act: Vul nu de wekelijkse vragenlijsten in" oid ipv "u-can-act vragenlijstonderzoek"? (wat trouwens één woord is).
Het subject is nu `u-can-act`. Waar ik al bang voor was, de mail wordt inderdaad gecollapsed in gmail:
![u-can-act vragenlijst onderzoek - frank blaauw gmail com - gmail 2017-12-12 11-57-20](https://user-images.githubusercontent.com/617574/33881002-bcefe01c-df33-11e7-87cb-e6feaa3f0fc3.png)

Ik heb nu een klein ref ding aan het einde toegevoegd:
![mailcatcher 3 2017-12-12 11-58-58](https://user-images.githubusercontent.com/617574/33881042-dc2419f8-df33-11e7-8a53-b6c531be1a72.png)
dan gaat het sowieso goed.

> Of willen we juist de dynamische regel in het subject hebben? bv: "u-can-act: Hoi Koos ff invullen die hap"
Nee ik had het er gisteren met nick over, het wordt gewoon iets constants.

Reply:
![u-can-act - frank blaauw gmail com - gmail 2017-12-12 12-19-37](https://user-images.githubusercontent.com/617574/33881932-bd31414e-df36-11e7-91e3-0134ac4a97a6.png)